interpreter/interpreter_test.go

```diff
@@ -165,11 +167,9 @@ func TestEval(t *testing.T) {
             six = six()
             nine = nine()
 
-            answer = (not (fortyTwo() == six * nine)) or fail()
+            not (fortyTwo() == six * nine) or fail()
 			`,
 			want: []values.Value{
-				values.NewFloat(6.0),
-				values.NewFloat(9.0),
```

Comment:
Why did this test need to change?

Reply:
Variable declarations aren't side effects so I got rid of them to reduce confusion.